base/mmap.jl

```diff
@@ -57,9 +57,9 @@ grow!(::Anonymous,o::Integer,l::Integer) = return
 function grow!(io::IO, offset::Integer, len::Integer)
     pos = position(io)
     filelen = filesize(io)
-    if filelen < offset + len
-        write(io, Base.zeros(UInt8,(offset + len) - filelen))
-        flush(io)
+    failure = ccall(:ftruncate, Int16, (Int, Int), fd(io), offset+len)
```

Comment:
return value should be `Cint`, first argument should be `Cint`, second argument should be `Coff_t`